core/htmlsanitizer.class.inc.php

```diff
@@ -34,32 +34,36 @@ abstract public function DoSanitize($sHTML);
 
 	/**
 	 * Sanitize an HTML string with the configured sanitizer, falling back to HTMLDOMSanitizer in case of Exception or invalid configuration
+	 *
 	 * @param string $sHTML
+	 * @param string $sConfigKey
```

Comment:
Maybe add examples of conf keys?
```suggestion
	 * @param string $sConfigKey eg. 'html_sanitizer', 'svg_sanitizer'
```